package.json

```diff
@@ -74,15 +74,15 @@
     "eslint-scope": "^8.0.1",
     "espree": "^10.0.1",
     "husky": "^9.1.7",
-    "jiti": "^2.4.2",
+    "jiti": "^2.5.1",
     "lodash": "^4.17.21",
     "markdownlint-cli": "^0.43.0",
     "npm-package-json-lint": "^8.0.0",
     "npm-run-all2": "^7.0.1",
     "prettier": "^3.4.1",
     "release-it": "^17.2.0",
     "tsup": "^8.5.0",
-    "typescript": "^5.8.3",
+    "typescript": "~5.9.2",
```

Comment:
Let's stick with caret. We can be more aggressive with dependency updates in this repo.